### PR TITLE
Added ability to pass chromeOptions command line arguments

### PR DESCRIPTION
**Pull request by pturchinetz**
You can use -DchromeOptions="start-maximized" to open chrome windows in a maximized state

### CODE REVIEW

File: src/main/java/com/dougnoel/sentinel/webdrivers/WebDriverFactory.java [hunk 1]
```diff
@@ -157,11 +157,15 @@ private static void setChromeDownloadDirectory(ChromeOptions options) {
...
      * Creates a ChromeDriver. Makes it headless if the -Dheadless flag is set.
+     * Can pass additional arguments with the -DchromeOptions flag, such as -DchromeOptions="start-maximized" to open all browser windows maximized.
      * @return WebDriver ChromeDrvier
      */
     private static WebDriver createChromeDriver() {
     	var chromeOptions = new ChromeOptions();
     	setChromeDownloadDirectory(chromeOptions);
+        String commandlineOptions = Configuration.toString("chromeOptions");
+        if (commandlineOptions != null)
+            chromeOptions.addArguments(commandlineOptions);
```
Comment (dougnoel) [inline src/main/java/com/dougnoel/sentinel/webdrivers/WebDriverFactory.java:168]: Add a unit test for this. Replace the chrome brad browsertest in WebDriverFactoryTests as that also exists in WebDriverFctoryBadBrowserNameTests.

**Review by dougnoel — CHANGES_REQUESTED** [review]

File: src/main/java/com/dougnoel/sentinel/webdrivers/WebDriverFactory.java (hunk 1 above)
Comment (pturchinetz) [reply src/main/java/com/dougnoel/sentinel/webdrivers/WebDriverFactory.java:168]: Done!

File: src/test/java/com/dougnoel/sentinel/webdrivers/WebDriverFactoryTest.java [hunk 2]
```diff
@@ -69,9 +69,15 @@ public void malFormedUrlTest() {
...
 	}
 	
-	@Test(expected = org.openqa.selenium.WebDriverException.class)
-	public void createCustomChromeDriverTest() {
-		System.setProperty("chromeBrowserBinary","fake/path");
+	@Test
+	public void createChromeOptionsChromeDriver() {
+		Configuration.update("chromeOptions", "start-maximized");
+		WebDriverFactory.instantiateWebDriver();
+	}
```
Comment (dougnoel) [inline src/test/java/com/dougnoel/sentinel/webdrivers/WebDriverFactoryTest.java:76]: The last line can be put in the after step.
```suggestion
	@Test
	public void createChromeOptionsChromeDriver() {
		Configuration.update("chromeOptions", "start-maximized");
		WebDriverFactory.instantiateWebDriver();
		var js = (JavascriptExecutor)Driver.getWebDriver();
		assertSame("Expecting window to be maximized.", "false", js.executeScript("return document.hidden").toString());
		WebDriverFactory.quit();
	}
```

File: src/test/java/com/dougnoel/sentinel/webdrivers/WebDriverFactoryTest.java [hunk 3]
```diff
@@ -69,9 +69,15 @@ public void malFormedUrlTest() {
...
+
+	@Test
+	public void createMultipleChromeOptionsChromeDriver() {
+		Configuration.update("chromeOptions", "--start-maximized --incognito");
 		WebDriverFactory.instantiateWebDriver();
```
Comment (dougnoel) [inline src/test/java/com/dougnoel/sentinel/webdrivers/WebDriverFactoryTest.java:81]: There's no assertion here. Se the above comment for how to add an test.

**Review by dougnoel — CHANGES_REQUESTED** [review]
I also thought we were removing the comments from the WebDriverFactoryTests.java file.

**Comment by pturchinetz** [comment]
> 

That is in the attribute changeset

**Review by dougnoel — APPROVED** [review]

**Pull request merged**